awscli/customizations/paginate.py

```diff
@@ -35,6 +35,9 @@
 
 logger = logging.getLogger(__name__)
 
+# There are some services where the pagination params need to be unhiden
+PUBLIC_PAGINATION_TOKENS = {'route53': ['start-record-name']}
```

Comment:
This may need to get updated if we ever want to support operation level filtering, but it is fine for now.